programmers/level01.평균_구하기/SSSOy.java

```diff
@@ -0,0 +1,10 @@
+class Solution {
+    public double solution(int[] arr) {
+        double answer = 0;
+        
+        for(int i : arr) 
+            answer += i;
+        
+        return answer / arr.length;
```

Comment:
```suggestion
        return Arrays.stream(arr).reduce(0.0, Double::sum) / arr.length;
```
이런식으로 표현할 수 있습니다!
그런데 Double로 이렇게 형 변환이 될지는 확신을 못하겠네요.. ㅎㅎ

Reply:
와 알려주셔서 감사합니다!!